### ISSUE TITLE
Firefox: Answer tried to set recv when offer did not set send

### ISSUE THREAD

**Issue by jsmouret** [pion/webrtc#1700]
### Your environment.
  - Version: pion/webrtc v3.0.11
  - Browser: Firefox 78.8.0esr
  - Other Information 

### What did you do?

Using firefox, when creating an offer with:

```
peer.addTransceiver("video", { direction: "recvonly" })
```

and replying in pion using:
``` 
peer.AddTrack(track)
```

Firefox complains that:
```
Answer tried to set recv when offer did not set send
```

### What did you expect?

No error.

### What happened?

I guess something is wrong in rtptransceiver.go / setSendingTrack

Thanks

**Comment by jsmouret** [comment]
with pion 3.0.14, firefox doesn't complain but AddTrack returns:

**Comment by jsmouret** [comment]
fixed on v3.0.18
